### ISSUE TITLE
Syntax error when  loading Dockerfile

### ISSUE THREAD

**Issue by etcadinfinitum** [bdougie/invite-based-on-comment#2]
Hi Brian,

I am working on setting up this Action in an organization repository. The repository has been created explicitly for a git workshop that my ACM chapter is hosting, and we expect 40+ attendees with 2 facilitators available to manage repository access; needless to say, automating repository invitations is a desirable feature that we would like for this use case.

However, when I try to run the action, I get a Docker error, which appears to be related to the script:

```
Build container for action use: '/home/runner/work/_actions/bdougie/invite-based-on-comment/master/Dockerfile'.
/usr/bin/docker build -t e87b52:31aac48af5584fca8d523828ad7d5568 "/home/runner/work/_actions/bdougie/invite-based-on-comment/master"
Error response from daemon: failed to parse Dockerfile: Syntax error - can't find = in "me\" or \".invite". Must be of the form: name=value
Sending build context to Docker daemon  7.168kB

##[warning]Docker build failed with exit code 1, back off 8.347 seconds before retry.
```

See [the action report](https://github.com/UWB-ACM/FancyCalculator/runs/468652650?check_suite_focus=true) for more details.

Based on what I know about how actions are initiated and set up, I don't expect that this issue is related to authentication problems; instead, it appears to be a syntax issue in [the Ruby script](https://github.com/bdougie/invite-based-on-comment/blob/master/action.rb). Would you be willing to take a look in the next day or two?

[The workflow .yml file for the repository is here.](https://github.com/UWB-ACM/FancyCalculator/blob/master/.github/workflows/invite.yml)

**Comment by bdougie** [comment]
Hi @etcadinfinitum, sorry I missed this. This action was created before there were proper ways to present error messages. I will try to push an update for that.

regarding your error, I am interpreting that the comment only included `.invite` and omitted either `me` or the `user's handle.`

I am unable to see where the comment was left to confirm.

**Comment by etcadinfinitum** [comment]
https://github.com/UWB-ACM/FancyCalculator/issues/60

**Comment by etcadinfinitum** [comment]
This action didn't end up working for my use case and I added all of the contributors who commented manually. You can see for yourself in the linked issue that many of the users used the correct syntax you specified; it seems to be an error with the Ruby script as opposed to user invocation.

**Comment by jmertic** [comment]
Hey @etcadinfinitum - I know this is probably late, but I did fix this in #5 .

**Comment by etcadinfinitum** [comment]
> Hey @etcadinfinitum - I know this is probably late, but I did fix this in #5 .

Awesome, thanks @jmertic!
